src/fallbacks.jl

```diff
@@ -66,6 +71,10 @@ getpoint(t::AbstractMultiPolygonTrait, geom) = flatten((p for p in getpoint(r))
 npatch(t::AbstractPolyhedralSurfaceTrait, geom)::Integer = ngeom(t, geom)
 getpatch(t::AbstractPolyhedralSurfaceTrait, geom) = getgeom(t, geom)
 getpatch(t::AbstractPolyhedralSurfaceTrait, geom, i::Integer) = getgeom(t, geom, i)
+getpoint(t::AbstractPolyhedralSurfaceTrait, geom) = flatten((p for p in getpoint(ls)) for ls in getgeom(t, geom))
```

Comment:
A surface consists of patches (Polygons), not of linestrings (ls).

Reply:
Shouldn't this just work anyway because the flattening is recursive?